### ISSUE TITLE
Separate config for each useSWRV() invocation on the same key

### ISSUE THREAD

**Issue by ivos** [Kong/swrv#239]
On version `1.0.0-beta.8`.

Calling `useSWRV` on the same `key` twice with different configs seems not to honor them.

Example:

```
const { data: refreshingData } = useSWRV('/foos/42', fetch)
const { data: stableData } = useSWRV('/foos/42', fetch, { revalidateOnFocus: false })
```

Here both `refreshingData` and `stableData` gets refreshed on focus. Expected behavior would be to refresh `refreshingData` on focus, but leave `stableData` as-is.

(Why: Stable data serves as the basis for user-editable form. It must not change while the user is in the middle of editing it. Refreshing data, on the other hand, is read-only and it should update asap when there's been a change in the meantime.)

**Comment by darrenjennings** [comment]
just dropping a note that I wont' be getting to this soon, but tagged "help wanted" and appreciate any PR's!

**Comment by ivos** [comment]
So I wrote a test exposing this issue. A couple of log statements later I realized the configs in fact **are** separate and honored correctly.

The issue with the code above is that the same key means the newly refreshed value is replaced in the cache for the key which is shared between both invocations and therefore even the `stableData` then gets refreshed although it didn't cause the fetch.

The solution here is to use separate keys. One has to have a custom fetcher then to use the same URL even though the keys are different.

I'm offering the test, if you find it useful: https://github.com/Kong/swrv/pull/276.

This being a bit tricky issue (not only me thought it was a bug), it might be a good idea to smh try to explain this in the docs to preempt future confusions.
